xitorch/_tests/test_linop_fcns.py

```diff
@@ -563,7 +563,8 @@ def solvefcn(amat, bmat, emat):
 
     ax = LinearOperator.m(amat).mm(x)
     xe = torch.matmul(x, torch.diag_embed(emat, dim2=-1, dim1=-2))
-    assert torch.allclose(ax - xe, bmat)
+    # temporary solution until better convergence of gmres is obtained
+    assert torch.allclose(ax - xe, bmat, rtol=1e-1)

```

Comment:
does it have to be 1e-1? can you make rtol lower by increasing atol?

Reply:
I can probably make atol big, for example, 1e-1, and use the default value for rtol, how does that sound?

Reply:
the deviation is about 2 decimal places

Reply:
How about make an `if method == "gmres"`? My worry is it allows the other method to decrease their accuracy

Reply:
Yes that is a better solution for now, fixed it in f24fe48